sqlite-via-api-client.py

```diff
@@ -22,8 +22,10 @@
 
 # Keep polling for changes, making sure to persist the cursor atomically to saving the changes
 while True:
-    r = s.post("http://localhost:8080/api/events", json={"cursor": cursor}).json()
-    cursor, rows = r["cursor"], r["events"]
+    r = s.post("http://localhost:8080/api/events", json={"cursor": cursor})
+    r.raise_for_status()
+    data = r.json()
+    cursor, rows = data["cursor"], data["events"]
```

Comment:
Self Review:

`r.raise_for_status` returns a more intuitive error then a json decode error and was required at one point when I didn't have demo app working correctly.